src/common/serverinfo.ts

```diff
@@ -70,15 +61,31 @@ function getServerInfo(this: RippleAPI): Promise<GetServerInfoResponse> {
   })
 }
 
-async function getFee(this: RippleAPI, cushion?: number): Promise<string> {
+async function getFee(
+  this: RippleAPI,
+  cushion?: number,
+  maximum: number | BigNumber | null = 1
```

Comment:
`getFee`'s arguments (including the new default maximum of 1) are not documented
https://github.com/ripple/ripple-lib/blob/51197a1c6848283abca8b34ff4f9a4f0ffe1deaa/docs/index.md#getfee

Reply:
Done